tests/framework/installer/ceph_manifests.go

```diff
@@ -2042,7 +2050,7 @@ func (m *CephManifestsMaster) GetBlockPoolStorageClass(namespace string, poolNam
 }
 
 // GetFilesystem returns the manifest to create a Rook filesystem resource with the given config.
-func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount int) string {
+func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount int, activeStandbyCount int) string {
```

Comment:
```suggestion
func (m *CephManifestsMaster) GetFilesystem(namespace, name string, activeCount, activeStandbyCount int) string {
```

Reply:
@leseb I need to make the same change in `tests/framework/installer/ceph_manifests_v1.1.go` right?

Reply:
No.